Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -344,6 +360,56 @@ extension SPNoteListViewController {
 
         emptyTrashButton.isEnabled = isTrashOnScreen && isNotEmpty
     }
+
+    /// Delete selected notes
+    ///
+    @objc
+    func trashSelectedNotes() {
+        guard let notes = tableView.indexPathsForSelectedRows?.compactMap({ notesListController.object(at: $0) as? Note }) else {
+            return
+        }
+
+        delete(notes: notes)
+        setEditing(false, animated: true)
+    }
+
+    /// Setup Navigation toolbar buttons
+    ///
+    @objc
+    func configureNavigationToolbarButton() {
+        guard let trashButton = trashButton,
+              let addButton = addButton else {
+            return
+        }
+        let flexibleSpace = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+        let actionButton = isEditing ? trashButton : addButton
+        setToolbarItems([flexibleSpace, actionButton], animated: true)
+    }
+
+    @objc
+    func selectAllWasTapped() {
+
+        if notesListController.numberOfObjects == tableView.indexPathsForSelectedRows?.count {
+            tableView.deselectAllRows(inSection: 0, animated: false)
```

Comment:
Please bear in mind you can use `.zero` everywhere there's a `0`.

Yeah it's definitely the same, buuut feels a whole lot less like magic numbers 🔥